scripts/ci/dependency_check.bat

```diff
@@ -9,8 +9,7 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 del baseline_deps.txt
 
 Rem Install everything from our repository first.
-.\scripts\install_full.bat
-if %errorlevel% neq 0 exit /b %errorlevel%
+call .\scripts\install_full.bat
```

Comment:
what happens with `call` if install_full.bat returns non-zero exit code?

Reply:
the script won't fail as all it is going to do, is `pip install`

Reply:
should we still keep `if %errorlevel% neq 0 exit /b %errorlevel%`, just in case?

Reply:
I could do so, but it's not necessary I think

Reply:
Adding it back for further modification of ./scripts/install_all.bat